scout/server/blueprints/variants/static/form_scripts.js

```diff
@@ -113,3 +147,53 @@ function enableDismiss(){
     btnElem.disabled = true;
   }
 }
+
+
+// Update chromosome position
+function updateCoordinateFields(element) {
```

Comment:
What does this function do? I think it's very useful if:

- if the user enters values in the chrom_pos:
  - The chromosome select is updated
  - start is updated (with eventual padding)
  - end is updated (with eventual padding)

What I don't think is useful (at least for the subsequent filtering):
- Update chrom_pos  if a user modifies chrom and cytobands, also because chrom_pos doesn't update if you modify chromosome, start and stop